client/src/App.test.tsx

```diff
@@ -3,10 +3,8 @@ import {render, screen} from '@testing-library/react';
 import App from './App';
 import {BrowserRouter} from "react-router-dom";
 
-test('test renders main page', () => {
-    render(
-        <BrowserRouter>
-            <App/>
-        </BrowserRouter>
-    );
+test('renders learn react link', () => {
+  // render(<App />);
+  // const linkElement = screen.getByText(/learn react/i);
+  // expect(linkElement).toBeInTheDocument();
```

Comment:
a bit confused about this change

Reply:
@htluu-ucsd

Reply:
i just wanted to run my own tests, can just be reverted

Reply:
it seems like it's the code from `start-react-app` so it probably got mixed into it somehow. can you revert it? thx!